python/cudnn/gemm/cutedsl/dense/amax/api.py

```
@@ -391,30 +437,53 @@ def gemm_amax_wrapper_sm100(
-    n, _, l = b_tensor.shape
-    c_tensor = None
-    if c_major == "m":
-        c_tensor = torch.empty_strided((m, n, l), (1, m, m * n), dtype=c_dtype, device=a_tensor.device)
-    elif c_major == "n":
-        c_tensor = torch.empty_strided((m, n, l), (n, 1, m * n), dtype=c_dtype, device=a_tensor.device)
-    else:
+    framework = detect_framework(a_tensor)
+    c_dtype = _convert_to_cutlass_data_type(c_dtype)
+    acc_dtype = _convert_to_cutlass_data_type(acc_dtype)
+
+    m, _, l = get_shape(a_tensor)
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)